### PR TITLE
Tests/dates

### PR DESCRIPTION
**Pull request by aberger3647**
## Description

Wrote tests for `getDaysBetweenDates()`, which is used in multiple places across the app.

## Related Issue

#11 , #13 

## Type of Changes

Unit test

## Testing Steps / QA Criteria

Run `npm test`

### CODE REVIEW

**Comment by aberger3647** [comment]
The tests all pass locally! Why don't they pass here?

File: src/utils/dates.test.js
Comment (bdanzer) [inline src/utils/dates.test.js:1]: great job on adding these tests!

File: src/components/ListItem.jsx [hunk 1]
```diff
@@ -56,6 +76,10 @@ export function ListItem({ item, listPath }) {
 		}
 	}
 
+	useEffect(() => {
```
Comment (bdanzer) [inline src/components/ListItem.jsx:79]: I put a comment about this useEffect in the other PR but there is a way to do this without useEffect or setUrgency

**Review by bdanzer — APPROVED** [review]

**Review by jtkabenni — APPROVED** [review]
Looks great!

**Pull request merged**